angular-hometask/src/app/core/breadcrumbs/breadcrumbs.component.scss

```diff
@@ -2,9 +2,39 @@
 
 :host {
   display: flex;
-  margin: 0 5rem;
-  p {
-      margin: 0.5rem 0 0;
+  margin:0 5rem;
+  .breadcrumb-container {
+    display: flex;
+    width: 100%;
+    padding: 0;
+    list-style: none;
```

Comment:
I saw u use ul, better solution here just reset this style in root style file

Reply:
Or better create style.scss for reset and import

Reply:
Done 31de302e4b5ceee0e7b55c28323d998a430679d7